piet-gpu/shader/kernel4.comp

```diff
@@ -27,6 +27,8 @@ layout(rgba8, set = 0, binding = 2) uniform writeonly image2D image;
 #include "ptcl.h"
 #include "tile.h"
 
+#define BLEND_STACK_SIZE 4
```

Comment:
Will you lift this restriction before replacing FillMask?

Reply:
I think so, but I'm not 100% sure when. On my AMD 5700 XT card, the static limit can be increased quite a bit without seriously impacting performance (utilization goes down as register pressure goes up). Using a scratch buffer in global memory can be done in two stages - malloc-only (using an atomic bump counter, as is currently done for dynamic allocations), and a second malloc/free stage.

Reply:
Can the FillMask stopgap co-exist with the new clipping? I'd hoped not to have hard limits such as this when adding piet-gpu as a Gio backend. I'll probably take a stab at #40, but I think this limit is worse.

Reply:
Thank you very much. If you decide to leave it, I'll take a look after #23, #38, #40 after which Gio has all it needs for now. The existing Gio renderer tests can be re-used and expanded to cover piet-gpu, and I'll be much more comfortable working on enhancements and bugs.